specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -728,25 +736,27 @@
 					"$ref": "./examples/SearchServiceCreateSkillset.json"
 				}
 			},
-			"description": "Creates a new Cognitive Search skillset.",
+			"description": "Creates a new Cognitive Search skillset under the search service.",
 			"externalDocs": {
-				"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
+				"url": "https://docs.microsoft.com/rest/api/searchservice/create-skillset"
 			},
-			"parameters": [{
-				"name": "skillset",
-				"in": "body",
-				"required": true,
-				"schema": {
-					"$ref": "#/definitions/Skillset"
+			"parameters": [
+				{
+					"name": "skillset",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"$ref": "#/definitions/Skillset"
+					},
+					"description": "The definition of skills."
```

Comment:
Please use the same description for this parameter as the one for the CreateOrUpdate operation.

Reply:
Fixed